cobigen-templates/templates-devon4j/src/main/templates/crud_ngrx_client_app/templates/cobigen.properties

```diff
@@ -0,0 +1 @@
+relocate:../../../workspaces_vs/devon4ng-ngrx-cobigen-template/${cwd}
```

Comment:
Please change the destination paths. For devon4ng is `relocate:../../devon4ng-application-template/${cwd}`

Reply:
Shouldn't it be `relocate:../../devon4ng-ngrx-template/${cwd}` since we are using the [devon4ng-ngrx-template](https://github.com/devonfw/devon4ng-ngrx-template) as the base template?

Reply:
Yes, sorry, it was an example. You are right.